packages/app/src/runner/aut-iframe.ts

```diff
@@ -39,6 +38,10 @@ export class AutIframe {
     this._showContents(blankContents.initial())
   }
 
+  showInitialBlankContentsCT () {
```

Comment:
Is it worth updating `showInitialBlankContents` to be `showInitialBlankContentsE2E` or something? It seems like we have pretty explicit naming in the other areas where we bifurcate based on testing type.